packages/web-components/src/styles/states/index.ts

```diff
@@ -180,6 +180,18 @@ export const importantState = css.partial`:is([state--important], :state(importa
  */
 export const informativeState = css.partial`:is([state--informative], :state(informative))`;
 
+/**
+ * Selector for the `marigold` state.
+ * @public
+ */
+export const marigoldState = css.partial`:is([state--marigold], :state(marigold))`;
+
+/**
+ * Selector for the `neutral` state.
+ * @public
+ */
+export const neutralState = css.partial`:is([state--neutral], :state(neutral))`;
```

Comment:
One minor adjustment we might want to do later is make it clear that these are themes or color states. `marigoldColorState` or `marigoldTheme`. Same with `brandState` above. We should probably wait until we have more themes and/or the new semantic tokens system normalizes.

Reply:
Good suggestion for a follow up! It'll make it clearer not just for colors but also other states that have more generic names, such as "error" or "neutral."